src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/acl_editor.cljs

```diff
@@ -45,14 +45,18 @@
               :spellCheck false
               :value (:email acl-entry)
               :onChange #(swap! state assoc-in [:non-project-owner-acl-vec i :email] (.. % -target -value))}])
-          (style/create-identity-select
-           {:ref (str "acl-value" i)
-            :style {:display "inline-block" :width 200 :height 33 :marginLeft "1rem" :marginBottom 0}
-            :disabled (= (:email acl-entry) (-> @utils/google-auth2-instance (.-currentUser) (.get) (.getBasicProfile) (.getEmail)))
-            :value (:accessLevel acl-entry)
-            :onChange #(swap! state assoc-in [:non-project-owner-acl-vec i :accessLevel]
-                              (.. % -target -value))}
-           access-levels)
+          (let [user-access-level-index (.indexOf (to-array access-levels) user-access-level)
```

Comment:
Yeah, I hope that one day we don't even have the idea of access levels, but for now I should add a comment near `access-levels` stating that a piece of code relies on it being in descending order.